practice-two/src/style/sections/project.css

```diff
@@ -0,0 +1,158 @@
+.project__contents {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 35px;
+  margin-top: 20px;
+  margin-bottom: 50px;
+}
+
+.project__item {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+}
+
+.project__item-description {
+  display: flex;
+  flex-direction: column;
+}
+
+.project__item-title {
+  text-align: center;
+  font-size: var(--font-size-xs);
+  font-style: var(--font-style-default);
+  font-weight: var(--font-weight-primary);
+  line-height: 24px;
+  text-transform: uppercase;
+}
+
+.project__item-author::before {
+  content: "by";
+  display: block;
+  line-height: var(--line-height-2xlg);
+}
+
+.project__item-author {
+  text-align: center;
+}
+
+.project-btn {
+  margin-top: 10px;
+
+  &:hover {
+    background-color: var(--color-primary);
+  }
+}
+
+.project__header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 29px;
+}
+
+.project__header-text {
+  text-align: center;
+}
+
+.content__item-img {
+  background-color: var(--color-primary);
```

Comment:
This style only display when you hover over the image

Reply:
@huynhthikimtrung321 you overlooked my comment. Could you please review it again